CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,78 @@
+# Changelog 2.5
+
+## 2.5.0 - 2025-11-28
+
+### Added
+
+- Add support for Antrea Egress in hybrid mode. ([#7239](https://github.com/antrea-io/antrea/pull/7239), [@hongliangl])
+- Add IPv6 support for PacketCapture feature, including ICMPv6 handling and new CLI filter options. ([#7385](https://github.com/antrea-io/antrea/pull/7385), [@harshgdev])
+- Support arbitrary source or destination, allowing users to capture all ingress or egress traffic for a Pod by specifying only one endpoint. ([#7215](https://github.com/antrea-io/antrea/pull/7215), [@harshgdev])
```

Comment:
This is confusing here. I cannot tell that which feature is enhanced.

Reply:
"Enhance PacketCapture to support matching arbitrary source or destination"?

Reply:
Updated.